index.html

```diff
@@ -0,0 +1,312 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/774f0ae2db.js" crossorigin="anonymous"></script>
+  <link href="https://fonts.googleapis.com/css2?family=Sansita:wght@700&family=Spinnaker&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="css-reset.css">
+  <title>Document</title>
+</head>
+
+<body>
+  <nav class="nav-bar">
+    <div class="smashing-logo">
+      <img src="./images/logo--full.svg" alt="Smashing Logo">
+    </div>
+    <div class="nav-grow">
+      <img src="./images/articles.svg" alt="Articles">
+      <div class="nav-text">
+        <h2>Articles</h2>
+        <span>Design & development</span>
+      </div>
+    </div>
+    <div class="nav-grow">
+      <img src="./images/books.svg" alt="Books">
+      <div class="nav-text">
+        <h2>Books</h2>
+        <span>Physical & digital books</span>
+      </div>
+    </div>
+    <div class="nav-grow">
+      <img src="./images/events.svg" alt="Events">
+      <div class="nav-text">
+        <h2>Events</h2>
+        <span>Conferences & workshops</span>
+      </div>
+    </div>
+    <div class="nav-grow">
+      <img src="./images/jobs.svg" alt="Jobs">
+      <div class="nav-text">
+        <h2>Jobs</h2>
+        <span>Find work & employees</span>
+      </div>
+    </div>
+    <div class="nav-grow">
+      <img src="./images/membership.svg" alt="Membership">
+      <div class="nav-text">
+        <h2>Membership</h2>
+        <span>Webinars & early-birds</span>
+      </div>
+    </div>
+    <div class="search-button">
+      <div class="search-text">
+        <i class="fas fa-search"></i>&nbsp;&nbsp;&nbsp;<span>Topics</span>
+      </div>
+    </div>
+
+  </nav>
+
+  <main class="main-area">
+    <div class="main-articles">
+      <div class="center"></div>
+      <div class="article1"></div>
```

Comment:
- [ ] These, .article1, 2, 3, etc. Should be `<articles>`
- [ ] Remember to also add text to them.